### PR TITLE
feat: add aldhair-vera.md

### PR DESCRIPTION
**Pull request by BurningAl15**
### Description
Add aldhair-vera.md file

### Feeling
[//]: <> (Como te sientes con este PR? la solucion que entregas como te hace sentir?)
- [x] 🤙 Quick Fix
- [ ] 👌 Implented, Tested and Done
- [ ] 🤞 I really hope this works, please review it

### Ticket (Issue Tracking)
CV-13

### Scope
- [ ] 🐞 Bugfix (non-breaking changes to fix a bug)
- [x] 💚 Enhancement (non-breaking change to add/modify functionality to an existing feature)
- [ ] ⚡️ Feature (non-breaking change to add a new feature)
- [ ] ⚠️ Breaking change (change that is not backward compatible, changes to current functionality)
̌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Adicionales: M-J
 10. Juan Buitragǒ
 11. Giovanni García
 12. Piero Loza
-13. Nedes Vera
+13. [Aldhair Vera](profiles/aldhair-vera.md)
```
Comment (pablovem) [inline README.md]: Please remove changes in this file.

File: profiles/aldhair-vera.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+También me gusta programar tanto en el trabajo como en mi tiempo libre (donde aprendo sobre otros lenguajes, frameworks y engines).
+Me gusta probar nueva comida también, o experimentar creando comida por mi mismo (de momento, casi todos mis experimentos tienen muy buen sabor).
+---
+He estado trabajando en videojuegos por 4 años, haciendo juegos móviles, web, realidad virtual, juegos sencillos para PC usando Unity, C++ y OpenGL, Gamemaker.
+
+```
+
+
+## Social Media
+- [LinkTree] https://linktr.ee/aldhairvera
```
Comment (pablovem) [inline profiles/aldhair-vera.md]: Please fix this link.

**Review by pablovem — CHANGES_REQUESTED** [review]

File: profiles/aldhair-vera.md (hunk 2 above)
Comment (BurningAl15) [reply profiles/aldhair-vera.md]: Link fixed.

File: README.md (hunk 1 above)
Comment (BurningAl15) [reply README.md]: Change removed in file.

**Review by pablovem — APPROVED** [review]
LGTM!

**Review by saradvr — APPROVED** [review]

**Pull request merged**